workspaces/arborist/test/node.js

```diff
@@ -2753,6 +2755,7 @@ t.test('overrides', (t) => {
           name: 'baz',
           version: '1.0.0',
           pkg: {
+            version: '1.0.0',
```

Comment:
the package comparison here made me do this. Does the version belong in the pkg? Or the outer part? Or both? https://github.com/npm/cli/pull/8089/files#diff-4e00e62cbcb655399ea5f1c3869d23ca15b59e69fccf95e16a60a9075f6de2deR363

Reply:
Got it, it really does belong in the pkg. When it is further outside of that, it's a convenience method